Source/WebCore/Modules/WebGPU/GPUBindGroupEntry.h

```diff
@@ -87,14 +87,18 @@ struct GPUBindGroupEntry {
             return false;
         });
     }
+    static bool equalSizes(const std::optional<GPUSize64>& a, const std::optional<GPUSize64>& b)
+    {
+        return (!a && !b) || (a && b && *a == *b);
+    }
```

Comment:
According to https://en.cppreference.com/w/cpp/utility/optional/operator_cmp `std::optional::operator==` will do this check that the emptiness of both values is the same.